src/FxCop/Desktop.Analyzers/Core/Security/DoNotUseInsecureCryptographicAlgorithmsAnalyzer.cs

```diff
@@ -130,30 +153,33 @@ public void AnalyzeNode(SyntaxNodeAnalysisContext context)
 
                 if (type.IsDerivedFrom(this._cryptTypes.DES, baseTypesOnly: true))
                 {
-                    rule = DoNotUseDESSpecificRule;
+                    rule = CreateCA5351DiagnosticDescriptor(owningParentName, _cryptTypes.DES.Name);
```

Comment:
This is trying to avoid to create descriptors for all problematic crypto algorithms. We will end up with 7~9 descriptors with nothing but a small piece of description is different in fields. Isn't this cleaner?